package.json

```diff
@@ -13,7 +12,7 @@
   },
   "homepage": "https://github.com/Elgg/Elgg",
   "scripts": {
-    "test": "karma start js/tests/karma.conf.js --single-run"
+    "test": "node_modules/karma/bin/karma start js/tests/karma.conf.js --single-run"
```

Comment:
Glad this worked. Kind of annoying, but I guess `npm test` abstracts away the boilerplate anyways so whatevs!